src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/historikk/BehandlingshistorikkService.kt

```diff
@@ -68,4 +69,34 @@ class BehandlingshistorikkService(
             ),
         )
     }
+
+    fun slettFritekstMetadataVedFerdigstillelse(behandlingId: BehandlingId) {
+        val relevanteHistorikkinnslag =
+            behandlingshistorikkRepository
+                .findByBehandlingIdAndStegIn(
+                    behandlingId,
+                    listOf(StegType.SEND_TIL_BESLUTTER, StegType.BESLUTTE_VEDTAK),
+                )
+
+        relevanteHistorikkinnslag.forEach { historikkinnslag ->
+            val oppdatertHistorikkinnslag = historikkinnslag.slettFritekstmetadata()
+            behandlingshistorikkRepository.update(oppdatertHistorikkinnslag)
+        }
+    }
+
+    fun Behandlingshistorikk.slettFritekstmetadata(): Behandlingshistorikk =
```

Comment:
Denne kan settes som private, eller flytte som en del av "Behandlingshistorikk"
```suggestion
    private fun Behandlingshistorikk.slettFritekstmetadata(): Behandlingshistorikk =
```